nf_core/pipeline-template/nextflow.config

```diff
@@ -68,10 +68,10 @@ params {
     validate_params                  = true
 
 }
-
+{% if modules %}
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
-
+{% endif %}
```

Comment:
why is the base config related to modules?

Reply:
It contains process parameters used in modules such as `cpus`, `memory`, labels... I think it makes sense to remove it if we don't use modules. Or do you think we should add a different feature `nextflow config` or similar?

Reply:
yes, let's have them as a feature called `process_configs`. (we used them also before we had modules, so I am thinking they might still be useful for general pipelines)

Reply:
After trying, I realised that we can't skip the `base.config` file alone because we use it in nf-core modules (such as [here](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/modules/nf-core/fastqc/main.nf#L29)). I think the best is to remove it together with components

Reply:
After some more thought, I remove the file but add the task, memory and cpus to `nextflow.config`, so these configuration can be reused for non standard pipelines: https://github.com/nf-core/tools/pull/3128/commits/8351e0c9326a28ab1f4eb2d5bdb947fd0d96c26d